doc/radosgw/notifications.rst

```diff
@@ -226,10 +286,12 @@ Response will have the following format:
 - User: name of the user that created the topic
 - Name: name of the topic
 - EndpointAddress: the push-endpoint URL
-- if endpoint URL contain user/password information, request must be made over HTTPS. If not, topic get request will be rejected.
 - EndpointArgs: the push-endpoint args
-- EndpointTopic: the topic name that should be sent to the endpoint (mat be different than the above topic name)
+- EndpointTopic: the topic name that should be sent to the endpoint (may be different than the above topic name)
+- HasStoredSecret: "true" if if endpoint URL contain user/password information. In this case request must be made over HTTPS. If not, topic get request will be rejected 
```

Comment:
s/if if/if;

Reply:
oh, if you meant "if and only if" use that, or "iff"

Reply:
(but this wouldn't really be a natural use of the mathematical iff)

Reply:
just a typo, will fix :-)